crates/router/src/consts.rs

```diff
@@ -136,3 +140,69 @@ pub const MAX_ALLOWED_AMOUNT: i64 = 999999999;
 //payment attempt default unified error code and unified error message
 pub const DEFAULT_UNIFIED_ERROR_CODE: &str = "UE_000";
 pub const DEFAULT_UNIFIED_ERROR_MESSAGE: &str = "Something went wrong";
+
+/// Regex for Identifying Card Network
+pub const CARD_NETWORK_DATA: Lazy<HashMap<common_enums::CardNetwork, CardNetworkPattern>> =
```

Comment:
we can reuse the CARD_NETWORK_DATA from common_utils consts

Reply:
Moved this into the is_cobadged_card()